test/functional/rpc_preciousblock.py

```diff
@@ -15,8 +16,7 @@ def unidirectional_node_sync_via_rpc(node_src, node_dest):
     while True:
         try:
             assert len(node_dest.getblock(blockhash, False)) > 0
-            break
```

Comment:
this `break` should not be removed

Reply:
Yes, that's why the CI timeout. I will resolve once I move the PR to ready

Reply:
Reverted break removed by mistake